guides/common/modules/con_host-group-settings-and-nested-host-groups.adoc

```diff
@@ -0,0 +1,111 @@
+[id="Host_Group_Settings_and_Nested_Host_Groups_{context}"]
+= Host Group Settings and Nested Host Groups
+
+A host group can define many settings for hosts, such as Lifecycle Environment, Content View, or Ansible roles that are available to the hosts.
+
+IMPORTANT: When you change the properties of a host group, the new settings do not propagate to the hosts assigned to the host group.
+Only Puppet class settings get updated on hosts after you change them in the host group.
+
+You can create a hierarchy of host groups.
+Aim to have one base level host group that represents all hosts in your organization and provides general settings, and then nested groups that provide specific settings.
+
+.Nested Host Group Hierarchy
+====
+You create a top-level host group named `Base` and two nested host groups named `Webserver` and `Storage`.
+The nested host groups are associated with multiple hosts.
+You also create host `custom.example.com` that is not associated with any host group.
+
+[cols="1,1,1"]
+|===
+|Top-level host group |Nested host group |Host
+
+.5+|*Base*
+
+This host group applies the `{EL}{nbsp}8.8` operating system setting.
+
+.2+|*Webserver*
+
+This host group applies the `linux-system-roles.selinux` Ansible role.
+
+|*webserver1.example.com*
+|*webserver2.example.com*
+
+.2+|*Storage*
+
+This host group applies the `linux-system-roles.postfix` Ansible role.
+
+|*storage1.example.com*
+|*storage2.example.com*
+
+|
+|*custom.example.com*
+
+|===
+
+In this hierarchy:
+
+* `Base` defines the settings common to all hosts. All hosts use {EL}{nbsp}8.8 as their operating system.
+* `webserver1.example.com` and `webserver2.example.com` use the `linux-system-roles.selinux` Ansible role.
+`storage1.example.com` and `storage2.example.com` use the `linux-system-roles.postfix` Ansible role.
+* `custom.example.com` inherits none of the Ansible roles defined by the host groups.
+====
+
+{Project} applies host settings in the following order when nesting host groups:
+
+* Host settings take priority over host group settings.
+* Nested host group settings take priority over parent host group settings.
```

Comment:
I'd move this above the first example.

Reply:
Speaking of attributes vs. properties, we're calling them a third thing here (settings) :slightly_smiling_face:

Reply:
Moved. I also changed all occurrences of 'attributes' and 'properties' to 'settings'. (There are some left in the procedures that follow, but I'll try to take care of that in https://github.com/theforeman/foreman-documentation/pull/2689.)